tools/EntityFramework.settings.targets

```diff
@@ -24,7 +24,8 @@ Project global pre-settings.
     <VersionReleasePrefix>0</VersionReleasePrefix>
     <VersionRelease>$(VersionReleasePrefix)-beta2</VersionRelease>
     <VersionReleaseName>Beta 2</VersionReleaseName>
-    <ToolingDotNetFrameworkVersion Condition="'$(VisualStudioVersion)' == '11.0' OR '$(VisualStudioVersion)' == '12.0' OR '$(VisualStudioVersion)' == '14.0'">v4.5</ToolingDotNetFrameworkVersion>
+    <ToolingDotNetFrameworkVersion Condition="'$(VisualStudioVersion)' == '11.0' OR '$(VisualStudioVersion)' == '12.0'">v4.5</ToolingDotNetFrameworkVersion>
+    <ToolingDotNetFrameworkVersion Condition="'$(VisualStudioVersion)' == '14.0'">4.5.1</ToolingDotNetFrameworkVersion>
```

Comment:
> @lajones Do you think it's it safe to update to 4.5.1?

better to to 4.5.2, [4.5 and 4.5.1 are no longer supported](https://blogs.msdn.microsoft.com/dotnet/2015/12/09/support-ending-for-the-net-framework-4-4-5-and-4-5-1/) only 4.5.2